contrib/dockerd-rootless.sh

```diff
@@ -35,25 +37,31 @@ if [ -z $rootlesskit ]; then
 	exit 1
 fi
 
-net=""
```

Comment:
could've continued using these as local variables, and set this one to the value of `DOCKERD_ROOTLESS_ROOTLESSKIT_NET`

(the use of the long all-caps env-vars is a bit "shouty" 😅)

Reply:
updated